src/extensions/default/InlineBorderRadiusEditor/BorderRadiusEditor.js

```diff
@@ -0,0 +1,818 @@
+
+define(function(require, exports, module) {
+    "use strict";
+
+    var KeyEvent           = brackets.getModule("utils/KeyEvent"),
+        PreferencesManager = brackets.getModule("preferences/PreferencesManager"),
+        StringUtils        = brackets.getModule("utils/StringUtils"),
+        Strings            = brackets.getModule("strings"),
+        Mustache           = brackets.getModule("thirdparty/mustache/mustache"),
+        BorderRadiusUtils   = brackets.getModule("utils/BorderRadiusUtils");
+
+       // tinycolor          = require("thirdparty/tinycolor-min")
+
+    /** Mustache template that forms the bare DOM structure of the UI */
+    var BorderRadiusTemplate = require("text!BorderRadiusEditorTemplate.html");
+    var DEFAULT_BORDER_RADIUS_VALUE = 0;
+    /**
+     * Box shadow editor control; may be used standalone or within an InlineBoxShadowEditor inline widget.
+     * @param {!jQuery} $parent  DOM node into which to append the root of the box-shadow editor UI
+     * @param {!{horizontalOffset: string, verticalOffset: string, blurRadius: string, spreadRadius: string, color: string}} values  Initial set of box-shadow values.
+     * @param {!function(string)} callback  Called whenever values change
+     */
+    function getIndividualValues(values){
+        var flag = true;
+        String.prototype.replaceAll = function(str1, str2, ignore)
+        {
+            return this.replace(new RegExp(str1.replace(/([\/\,\!\\\^\$\{\}\[\]\(\)\.\*\+\?\|\<\>\-\&])/g,"\\$&"),(ignore?"gi":"g")),(typeof(str2)==="string")?str2.replace(/\$/g,"$$$$"):str2);
+        } ;
+        var temp = values.replaceAll("px","A").replaceAll("%","B").replaceAll("em","C");
+        var finalValues = temp.replaceAll("A"," ").replaceAll("B"," ").replaceAll("C"," ").split(" ");
+
+        var empty = new Array();
+        for(var i=0;i<values.length;i++){
+            if(temp[i]==="A"){
+                empty.push("A");
+            }
+            else if(temp[i]==="B"){
+                empty.push("B");
+            }
+            else if(temp[i]==="C"){
+                empty.push("C");
+            }
+        }
+
+        if(finalValues.length!==empty.length+1){
+            return false;
+        }
+        var empty2 = new Array();
+        for(var j=0;j<empty.length;j++){
+           // console.log(typeof finalValues[j]);
+            //console.log(finalValues[j].match(/[a-z]/i));
+            if(isNaN(parseFloat(finalValues[j])))
```

Comment:
This loop needs some work.  You can probably do a better job of it with `Array.prototype.filter` to walk through the items and make sure that your value is defined.  Also, don't repeat code like `parseFloat(finalValues[j])`, save it in a variable and reuse.